src/components/editCqlLibrary/EditCqlLibrary.tsx

```diff
@@ -143,6 +160,28 @@ const EditCqlLibrary = () => {
     });
   };
 
+  const deleteDraftLibrary = async (id: string) => {
+    cqlLibraryServiceApi
+      .deleteDraft(id)
```

Comment:
Can you please add some test coverage for delete draft? Thanks!

Reply:
I think I forgot to add that file to the PR. I just added it thanks for pointing that

Reply:
@sb-prateekkeerthi great thanks!